runtime-management/runtime-management-impl/src/main/resources/eval.py

```diff
@@ -110,8 +110,25 @@ def main(self):
             try:
                 expr_result = eval(expression, smaller_context)
                 return_type = type(expr_result).__name__
-                if return_type not in ['str', 'int', 'bool']:
+
+                if return_type == 'range':
+                    expr_result = str(list(map(str, expr_result))).replace("\'", "\"")
+                    return_type = 'list'
+
+                elif return_type == 'list':
+                    expr_result = str(expr_result).replace("\'", "\"")
+
+                elif return_type == 'map':
+                    expr_result = str(list(expr_result)).replace("\'", "\"")
+                    return_type = 'list'
+
+                elif return_type == 'dict':
+                    expr_result = str(list(expr_result.keys())).replace("\'", "\"")
+                    return_type = 'list'
+
+                if return_type not in ['str', 'int', 'bool', 'list']:
```

Comment:
For dict, map and range the return type will always be overwritten 'str'. Did you forget to add add them to this list ?